.jenkins/pytorch/build.sh

```diff
@@ -58,6 +58,11 @@ if [[ "$BUILD_ENVIRONMENT" == *cuda* ]]; then
   nvcc --version
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *profile* ]]; then
```

Comment:
why can't this be `*coverage*`

Reply:
There is no particular coverage build. The current coverage test only captures Python coverage. In order to use lcov, we need a special build with the `--coverage` flag. My last PR wasn't able to capture that since there is no coverage specific build.